### DISCUSSION TITLE
Big Project vs Small Projects question

### DISCUSSION THREAD

**Discussion by abhuva** [Leantime/leantime#616] [Project Management]
Hello, i am quite new to this type of project management.
I am handling projects with round about 100k income in total (fundings) for a non-profit organization. Till now we used Excel and other stuff to manage them but i reached the point where this is not working anymore.
I have 3 big projects to manage which devide into roundabout 25 subprojects that are kind of self-contained.
I am having a couple contractors i work with that are part of the team and would get tasks assigned aswell as put their hours into the time-sheets.

My first aproach right now is to make one single project and start to define all the milestones and tasks for all together - i quickly realized that this might be not the best aproach.
My question is: Would it be better to make a project in leantime for each subproject i have - or is it fine to handle them alltogether.
What are the pros and cons?

Thanks for any input!

**Comment by marcelfolaron** [comment]
Hey. Do the subcontractos need access to all 25 subprojects? 
My fiurst inclination would be to split them up into the 3 big projects. But really it depends on the access each contractor would need.

**Reply by abhuva** [reply]
Thanks, thats a good aproach to think about (level of access i need) - basically yes, the contractors work on different subprojects, so they atleast need access to use the time-tracking and gets assigned tasks.
I guess i will find a middle-way and group similar projects together (that either share a lot of the subcontractors or other stuff), so its more granular than just 3 big ones, but not as overwhelming as having 25...
Thanks for the answer

**Reply by gloriafolaron** [reply]
@abhuva some time has gone by — curious as to how you solved this for your use case there?
